set.mm

```diff
@@ -22287,24 +22287,43 @@ primitive connectives (no defined terms) on the right-hand side.
   $}
 
   ${
-    $d x y ph $.  $d y ps $.  $d y ch $.
-    mobidv.1 $e |- ( ph -> ( ps <-> ch ) ) $.
-    $( Formula-building rule for the at-most-one quantifier (deduction form).
-       (Contributed by Mario Carneiro, 7-Oct-2016.)  Remove dependency on
-       ~ ax-6 , ~ ax-7 , ~ ax-12 by adapting proof of ~ mobid .  (Revised by
-       BJ, 26-Sep-2022.) $)
-    mobidv $p |- ( ph -> ( E* x ps <-> E* x ch ) ) $=
-      ( vy weq wi wal wex wmo imbi1d albidv exbidv df-mo 3bitr4g ) ABDFGZHZDIZF
-      JCQHZDIZFJBDKCDKASUAFARTDABCQELMNBDFOCDFOP $.
+    $d y x $.  $d y ph $.  $d y ps $.
+    $( Equivalence theorem for the at-most-one quantifier.  (Contributed by BJ,
+       7-Oct-2022.) $)
+    mobi $p |- ( A. x ( ph <-> ps ) -> ( E* x ph <-> E* x ps ) ) $=
+      ( vy wb wal weq wex wmo imbi1 alimi albi syl alrimiv exbi df-mo 3bitr4g
+      wi ) ABEZCFZACDGZRZCFZDHZBUARZCFZDHZACIBCITUCUFEZDFUDUGETUHDTUBUEEZCFUHSU
+      ICABUAJKUBUECLMNUCUFDOMACDPBCDPQ $.
   $}
 
   ${
     mobii.1 $e |- ( ps <-> ch ) $.
     $( Formula-building rule for the at-most-one quantifier (inference form).
        (Contributed by NM, 9-Mar-1995.)  (Revised by Mario Carneiro,
-       17-Oct-2016.) $)
+       17-Oct-2016.)  (Proof shortened by BJ, 7-Oct-2022.) $)
```

Comment:
Yes indeed, and it was not caught by `MM-PA> minimize_with *`. Are you requiring some changes ?

Reply:
No tag, as usual with minimize results.

Reply:
But as I wrote above, this shortening is not the result of an automatic minimization. This is a manual shortening (even if a straightforward one).

Reply:
I do not particularly care about the credit, since that shortening is straightforward, but that part of ~conventions you're quoting is not about when to put a tag or not.  It seemed to me that all shortenings which are not automatic have to be recorded by a "shortened" tag.

Reply:
`Shorter proofs are welcome, and any shorter proof we accept will be acknowledged in the theorem description.  However...`
In my eyes this is also about adding a tag.  The 'However' part gives guidelines when a proof is acceptably shorter and when not.  And your proof is not, because of (3).

And what do you think, are the tags about?  If they are really meant to acknowledge something and be a reward inciting people to find shorter proofs, there should be IMO a minimum intellectual height, before your name is attributed to a proof.

Finally there is my 9-yrs experience with Norm, and how he has handled my several hundred contributions.  I don't know whether this counts anything here, but I don't think he would have encouraged me or someone else to single out a common subproof, and then have your name attached to any proof benefiting from it.

Reply:
Ok, I removed the tag, because to be clearer than above, I couldn't care less about having my initials attached to that. The important information in "(Proof shortened by BJ, 7-Oct-2022.)" is "(Proof shortened, 7-Oct-2022.)".  So, "what do I think tags are about?"  I think they are at least as much about credits than about recording the date of a specific change (they existed before we could do that using git history, and still now, there is a world between reading something plain on a webpage and having to download and learn git, then clone the repository, and finally explore its history).

Maybe this one is not worth recording indeed.  One of the problems is that ~conventions is written in an often vague and imprecise style and you want to adhere to them more strictly than they were intended to (or at least than @david-a-wheeler intended to when he wrote most of them, I think). So I guess we'll have to make ~conventions more precise over time, in a step-by-step manner.

Reply:
Ok, lets re-evaluate the rules.  I am fine with it.

Reply:
Although the discussion seems to be closed (and I do not want to open it again), I think the discussion was a little bit academic. If  ~mobi had been added before (because it it interesting by itself) or by another one, then rule (3) could have been applied in a second step to minimize ~mobii...